server/src/main/java/org/elasticsearch/index/mapper/NumberFieldMapper.java

```diff
@@ -1055,6 +1060,24 @@ public static Query longRangeQuery(
         }
     }
 
+    private static final Map<NumberType, ToScriptField<SortedNumericDocValues>> INTEGRAL_TYPE_TO_SCRIPT_FIELDS;
```

Comment:
You can use `Map.of()` here

Reply:
Or maybe this should be a method on `NumericType`: `fielddataBuilder(String name)`?

Reply:
Second suggestion is great. Cleans up this bit of change substantially. I'm just sad I didn't think of it :)